setup.py

```diff
@@ -13,5 +13,13 @@
 #    limitations under the License.
 
 from setuptools import setup
+from Cython.Build import cythonize
+import numpy
 
-setup()
+setup(
+    install_requires=[
+        "numpy",
+    ],
+    ext_modules = cythonize(["./dwave/plugins/sklearn/nearest_neighbors.pyx"]), 
```

Comment:
```suggestion
    ext_modules = cythonize(["./dwave/plugins/sklearn/nearest_neighbors.pyx"], annotate=True), 
```
Cython annotations are _super_ helpful. We do some stuff with environment variables in some of our other packages (e.g. [dimod](https://github.com/dwavesystems/dimod/blob/28ac803092d157d614526c8b72d9d041df7cb31d/setup.py#L76)) but there's no harm in just turning it on always. At some point I'll get around to making a PR to dimod to just change that to always be `True`.